release-notes/3.12.7.md

```diff
@@ -0,0 +1,17 @@
+# Release notes for Gluster 3.12.6
+
+This is a bugfix release. The release notes for [3.12.0](3.12.0.md), [3.12.1](3.12.1.md), [3.12.2](3.12.2.md), [3.12.3](3.12.3.md), [3.12.4](3.12.4.md), [3.12.5](3.12.5.md), [3.12.6](3.12.6.md) , [3.12.7](3.12.7.md)contain a listing of all the new features that were added and bugs fixed in the GlusterFS 3.12 stable release.
```

Comment:
I don't think you need to add a link to this document i,e ` 3.12.7`

Reply:
usually I follow the existing format, I can remove it if needed